ocp_resources/catalog_source_config.py

```diff
@@ -1,35 +1,48 @@
-from ocp_resources.constants import PROTOCOL_ERROR_EXCEPTION_DICT, TIMEOUT_4MINUTES
+from ocp_resources.constants import PROTOCOL_ERROR_EXCEPTION_DICT
```

Comment:
please remove this resource; it was deprecated in 4.5
```
oc get CatalogSourceConfig
error: the server doesn't have a resource type "CatalogSourceConfig"

```

Reply:
Done